specifyweb/frontend/js_src/lib/components/Merging/CompareSubView.tsx

```diff
@@ -177,8 +173,6 @@ function MergeDialog({
     <MergeDialogContainer
       buttons={
         <>
-          {childrenLength > 1 && <ToggleMergeView />}
```

Comment:
You can make the check for when the button appears smarter, but certainly don't remove the check

https://github.com/specify/specify7/pull/3935#issuecomment-1743948761

Didn't you already fix something similar recently, was it for the main dialog, not the sub-view?

Reply:
It was this one, and we remembered that we kept the check box at some point but nobody could remember the reason ^^